docs/core/tools/project-json-to-csproj.md

```diff
@@ -151,6 +168,24 @@ Use the `TargetFrameworks` property to define your list of target frameworks. Us
 </PropertyGroup>
 ```
 
+### Microsoft.NETCore.App metapackage
+
+```json
+{
+  "dependencies": {
+    "Microsoft.NETCore.App": "1.0.0"
```

Comment:
FYI in PJ, `"Microsoft.NETCore.App": "1.1.0"` and `"Microsoft.NETCore.App": { "type": "platform", "version": "1.1.0" }` are not the same thing.